.github/workflows/lint.yaml

```diff
@@ -16,3 +17,48 @@ jobs:
       - name: Lint bash scripts
         run: |
           bashate --ignore E006 --verbose tools/*.sh openstack/novarc
+
+  check-commit-message:
+    name: Check Commit Message
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get sources
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Check Commit Message
+        run: |
+          set -e -u
```

Comment:
LGTM. Just a suggestion: I'd move this to a file so it can be run locally as well (e.g. ./git-commit-msg-lint.sh <begin-commit-sha>  <end-commit-sha>)

Reply:
Good point. I'll do that. Thanks!